package.json

```diff
@@ -41,6 +41,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "ember-auto-import": "^1.5.2",
     "ember-cli-babel": "^6.6.0",
+    "ember-getowner-polyfill": "https://github.com/ember-polyfills/ember-getowner-polyfill.git#aeb1f672ff2659b9077516c89b4569999520e72c",
```

Comment:
Why a SHA? For your `setOwner` patch?

Reply:
Yeah, I'll remove this commit once the PR gets merged. I wanted to make sure it works in our internal app first.